### DISCUSSION TITLE
[Feature request] Disable group by date in Favorites tab

### DISCUSSION THREAD

**Discussion by Tarnyd** [immich-app/immich#11243] [General]
I don't save that many photos to favorites, which makes my favorites tab look like this.
### Juli 2024
sun 13 jul. 2024
**1 photo**
### May 2024
mon 20 may 2024
**1 photo**

And so on. 

In essence only showing three photos on each screen, because that is all that fits with all the surrounding labels. 

I would like the option to show a grid of thumbnails without any labels with years or dates.

**Comment by mikebakke** [comment]
100% agree - I have some albums I imported where all the dates are the same and it's a much cleaner display than the split by date which is not usually relevant when sharing photos with others. As ever, the option to choose is the way.
